src/server.cpp

```diff
@@ -1631,14 +1631,15 @@ void Server::SendSpawnParticle(session_t peer_id, u16 protocol_version,
 	pkt << pos << velocity << acceleration << expirationtime
 			<< size << collisiondetection;
 	pkt.putLongString(texture);
-	pkt << vertical;
```

Comment:
backwards compatibility required

Reply:
How can I do that? Is there an example for backwards compatible additional data in a packet?

Reply:
For data in the middle of a packet that was removed, you need to keep a dummy value in place of that.
For new data at the end, the deserialization need to be wrapped into a `try {} catch(SerializationError &e) {};`